wiki/Tournaments/OMMT/4/en.md

```diff
@@ -107,6 +107,35 @@ The osu!mania Malaysia Tournament 4 is run by various community members.
 
 *Note: Maps with asterisk in its ordering list is considered to be MAD map. It is a twist from the organiser to make certain map unbannable, and in some occasion will force players to use modifier to play along with it. Each stage may have different kind of MAD twist, details are written for each stage.*
```

Comment:
there are no asterisks (`*`) for any of the MAD maps

Reply:
At the very end, should I add it to the first line instead?
I missed the SV2 in Finals though.

Reply:
Applied